website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -585,6 +585,10 @@ components: sources: host_metrics: {
 	}
 
 	telemetry: metrics: {
-		processed_events_total: components.sources.internal_metrics.output.metrics.processed_events_total
+		component_errors_total:               components.sources.internal_metrics.output.metrics.component_errors_total
+		component_discarded_events_total:     components.sources.internal_metrics.output.metrics.component_discarded_events_total
+		component_received_bytes_total:       components.sources.internal_metrics.output.metrics.component_received_bytes_total
```

Comment:
Just checking, is the `BytesReceived` event already emitted for this source somewhere? I didn't see it in a quick glance over `mod.rs`.

Reply:
Good point.

Reply:
Is it possible to emit `BytesReceived` in this PR since it's required for source instrumentation? Or is there a reason we're leaving it off?

Reply:
Since we don't actually receive "bytes" from a source, I'm not sure what to do about this one. We could emit it with zero bytes each time just to have the metric present.

Reply:
I like the idea of emitting an event with zero bytes but is it really worth it considering it has a cost (maybe small but still).

Reply:
I'm 👍 on emitting `0` like we do for `demo_logs` for consistency in reporting.